.teamcity/_self/projects/DesktopApp.kt

```diff
@@ -72,7 +61,7 @@ object E2ETests : BuildType({
 				export USE_HARD_LINKS=false
 
 				# Build app
-				yarn run build-desktop:app
+				cd desktop && yarn run build
```

Comment:
This was an interesting finding.

The setup:
- Root project is using yarn v3 (specified in `./yarnrc`)
- `desktop` is using yarn v1 (specified in `desktop/.yarnrc`)

The root project has a script called `build-desktop:app` that runs `cd desktop && yarn run build`. Which yarn version will it run? Turns out the answer is "the same version that started the command", in this case yarn v3.

Reply:
If you were to change `desktop` to use the `.yarnrc.yml` config instead of `.yarnrc` it should respect it